resources/sass/components/_person-widget.scss

```diff
@@ -0,0 +1,60 @@
+.ris-person {
+    $this: &;
+    min-height: 200px;
+    width: 112px;
+    text-decoration: none;
+    box-shadow: 0 1px 6px 0 $c-box-shadow3;
+    border-radius: 4px;
+    color: $c-black;
+    display: flex;
+    flex-direction: column;
+    justify-content: stretch;
+    &:hover {
+        #{$this}__icon,
+        #{$this}__title {
+            color: $c-red;
+        }
+    }
+    @include breakpoint($bp-mobile-max) {
+        flex-direction: row;
+        min-height: auto;
+        height: 80px;
+        width: 100%;
+    }
+    &__image {
+        height: 112px;
+        max-width: 112px;
+        width: 100%;
+        @include breakpoint($bp-mobile-max) {
+            height: 80px;
```

Comment:
is it related to https://github.com/nextsolx/ratsinformationssystem/pull/61/files?file-filters%5B%5D=.js&file-filters%5B%5D=.scss&file-filters%5B%5D=.vue#diff-9f4b6ce907271f891474c4ca76d6360eR21? if yes use variable